cypress/constants/selectors.ts

```diff
@@ -0,0 +1,35 @@
+export const SIGN_UP_SELECTOR_IDS = {
```

Comment:
Let's avoid plurals when possible. The following would work too:

```suggestion
export const SIGN_UP_SELECTOR = {
```

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/137/commits/862724755b491f06f91679079206c56067191a8b